ui/component/or-mwc-components/src/or-mwc-table.ts

```diff
@@ -595,4 +704,12 @@ export class OrMwcTable extends LitElement {
         return this._dataTable ? (this._dataTable.root as HTMLElement) : (this.shadowRoot!.querySelector('.mdc-data-table') as HTMLElement);
     }
 
+    protected getColumnWidth(index: number, columns?: string[] | TableColumn[], tableWidthPx?: number): string | undefined {
+        return undefined;
```

Comment:
Note for myself; maybe make this a function a bit more useful compared to `undefined`. (if possible)